Orange/widgets/data/owrank.py

```diff
@@ -671,6 +671,11 @@ def create_scores_table(self, labels):
         new_table.name = "Feature Scores"
         return new_table
 
+    @staticmethod
+    def sort_by_col(measures, def_sel):
+        cols = [i + 1 for i, m in enumerate(measures) if m.name in def_sel]
+        return cols[0] if cols else len(measures) + 1
```

Comment:
What is `def_sel`? Passed in are `selected_checks`.
This method is named `sort_by_col()` but it does no sorting (as opposed to `self.ranksView.sortByColumn()`. I'd rename it to `select_col`.
The method is used in a single place and is short enough to bear moving it to where it's used.